docs/src/components/ElevationLevel.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import theme from "../theme";
+
+export const LevelDiv = styled.div`
+  height: 40px;
+  width: 40px;
+  position: relative;
+  top: 25%;
+  left: 25%;
+  background-color: ${props => props.backgroundColor};
+  border: ${props => props.border};
+  box-shadow: ${props => props.boxShadow};
+`;
+export const Container = styled.div`
+  height: 80px;
+  width: 80px;
+  background-color: ${theme.orbit.paletteWhite};
```

Comment:
It looks better to me without this white background, the alternating table row colors aren't a problem IMO.

Reply:
I think the reason was when you hover on rows with white backgrounds. The contrast gets to be pretty small. But the contrast is pretty bad anyway, so I agree it looks better without it. Not so great either way. 🤷